### PR TITLE
Update compose.py documentation and remove old code for tensorflow1

### PR DESCRIPTION
**Pull request by jbkyang-nvi**

### CODE REVIEW

File: compose.py [hunk 1]
```diff
@@ -71,12 +71,12 @@ def start_dockerfile(ddir, images, argmap, dockerfile_name, backends):
...
 
-    # PyTorch, TensorFlow 1 and TensorFlow 2 backends need extra CUDA and other
+    # PyTorch, TensorFlow backends need extra CUDA and other
     # dependencies during runtime that are missing in the CPU-only base container.
     # These dependencies must be copied from the Triton Min image.
     if not FLAGS.enable_gpu and (
         ("pytorch" in backends)
-        or ("tensorflow1" in backends)
+        or ("tensorflow" in backends)
         or ("tensorflow2" in backends)
```
Comment (rmccorm4) [inline compose.py:80]: do we need the `tensorflow2` case as well?

Comment (jbkyang-nvi) [reply compose.py:80]: We do it for [build.py](https://github.com/triton-inference-server/server/blob/main/build.py#L2484) but I think yeah makes sense to remove

File: docs/customization_guide/compose.md [hunk 2]
```diff
@@ -74,32 +77,50 @@ script will extract components. The version of the `min` and `full` container
...
-Method 1 and 2 will result in the same composed container. Furthermore, `--image` flag overrides the `--container-version` flag when both are specified.
+Method 1 and 2 will result in the same composed container. Furthermore,
+`--image` flag overrides the `--container-version` flag when both are specified.
+
+Note:
+1. All contents in `/opt/tritonserver` repository of the `min` image will be
+ removed to ensure dependencies of the composed image are added properly.
+2. vLLM and TensorRT-LLM backends are currently not supported backends for
+`compose.py`. If you want to build additional backends on top of these backends,
+it would be better to [build it yourself](#build-it-yourself) by using
```
Comment (rmccorm4) [inline docs/customization_guide/compose.md:105]: @yinggeh @indrajit96 just double checking - do relative links like this work when creating the rendered HTML docs?

Comment (yinggeh) [reply docs/customization_guide/compose.md:105]: It works as long as the hashtag section exists in this document.

**Review by rmccorm4 — APPROVED** [review]

**Pull request merged**